### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by Aerindor-of-Arcane** [RubilaxXxx/ArsMagica2-5#17] [Q&A]
Hey @RubilaxXxx! do you have suggestions for a good discord server for active 1.7.10 modders? Or have one yourself?

**Comment by RubilaxXxx** [comment]
> Hey @RubilaxXxx! do you have suggestions for a good discord server for active 1.7.10 modders? Or have one yourself?

https://discord.gg/8cWbvu27

**Comment by RubilaxXxx** [comment]
Discord where you'll find most active modders in 1.7.10
